### ISSUE TITLE
[Notifier] [OvhCloud] “Invalid signature” for message with slashes

### ISSUE THREAD

**Issue by OneT0uch** [symfony/symfony#39836]
**Symfony version(s) affected**: 5.1.

**Description**  
"Invalid signature error" when i try to send a sms with ovh cloud which contain an url.

**How to reproduce**  
```
$message = "https://symfony.com/doc/current/index.html"
$notification = (new Notification($message))
    ->content('foo')
    ->importance(Notification::IMPORTANCE_HIGH);

$recipient = new AdminRecipient('insert email', 'insert phonenumber');

$this->notifier->send($notification, $recipient);
```

**Possible Solution** 
The json used as a signature is different from the json send by HTTPClient, resulting in an invalid signature

OvhCloudTransport.php (Line 79) 
Remove \JSON_UNESCAPED_SLASHES

` $toSign = $this->applicationSecret.'+'.$this->consumerKey.'+POST+'.$endpoint.'+'.json_encode($content, \JSON_UNESCAPED_SLASHES).'+'.$now;`

to

` $toSign = $this->applicationSecret.'+'.$this->consumerKey.'+POST+'.$endpoint.'+'.json_encode($content).'+'.$now;`

**Additional context**  
![image](https://user-images.githubusercontent.com/7141555/104611601-9a0b4100-5685-11eb-9834-71f2f5c9be22.png)

**Comment by OskarStark** [comment]
Can you provide a PR including a testcase?

**Comment by OskarStark** [comment]
cc @antiseptikk

**Comment by OskarStark** [comment]
While double checking this issue can you ensure the time of the machine is correct? To be more clear, can the calculated time delta be the problem of this issue? 🧐

**Comment by OneT0uch** [comment]
> While double checking this issue can you ensure the time of the machine is correct? To be more clear, can the calculated time delta be the problem of this issue? 🧐

Hi @OskarStark,
Time looks ok. 
I think the issue is unrelated because as i said it only occurred when i send a message with / character.  Otherwise it work great.
If it came from that, the signature would be invalid in both case? (Furthermore it's working when i edit the json encode)

**Comment by OskarStark** [comment]
Would you like to provide a PR?

**Comment by OneT0uch** [comment]
I open a pr with a test to show you the problem but i don't know how to solve that.

**Comment by OskarStark** [comment]
Closed by #39871
